pkg/constants/constants_windows.go

```diff
@@ -1,20 +1,20 @@
-package constants
-
-import (
-	"golang.org/x/sys/windows"
-)
-
-const (
-	AF_INET              = windows.AF_INET
-	AF_INET6             = windows.AF_INET6
-	PERF_MAX_STACK_DEPTH = 0x7f
-	CBitFieldMaskBit34   = 0x400000000
-	CAP_LAST_CAP         = 0x28
-	CAP_CHOWN            = 0
-	AF_UNIX              = windows.AF_UNIX
-	AF_NETBIOS           = windows.AF_NETBIOS
-	AF_IRDA              = windows.AF_IRDA
-	AF_BTH               = windows.AF_BTH
-	CGROUP2_SUPER_MAGIC  = 0x63677270
-	BPF_STATS_RUN_TIME   = 0
-)
+package constants
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+const (
+	AF_INET              = windows.AF_INET
+	AF_INET6             = windows.AF_INET6
+	PERF_MAX_STACK_DEPTH = 0x7f
+	CBitFieldMaskBit34   = 0x400000000
+	CAP_LAST_CAP         = 0x28
+	CAP_CHOWN            = 0
+	AF_UNIX              = windows.AF_UNIX
+	AF_NETBIOS           = windows.AF_NETBIOS
+	AF_IRDA              = windows.AF_IRDA
+	AF_BTH               = windows.AF_BTH
+	CGROUP2_SUPER_MAGIC  = 0x63677270
+	BPF_STATS_RUN_TIME   = 0
+)
```

Comment:
just fyi on this, I read through these patches very quickly, but you have somehow a NULL change here on this file? Maybe it's a whitespace change?

Reply:
https://github.com/cilium/ebpf/commit/2b8ce32946a5a38840e907894a4df55bd3aed815 could help

Reply:
Yes, I removed the CRLF. Which commit do you see that in, it is a new file?

Reply:
I see that in following commits:
- https://github.com/cilium/tetragon/pull/3488/commits/f345993fc0d43db51a92bb23cd59a4014dab90e3
- https://github.com/cilium/tetragon/pull/3488/commits/b274b117984ba63f6c633b577ee2388f0493f88c   (pkg/sensors/tracing/loader_windows.go)

please add the commit @lmb suggested above

Reply:
> I see that in following commits:
> 
>     * [f345993](https://github.com/cilium/tetragon/commit/f345993fc0d43db51a92bb23cd59a4014dab90e3)
> 
>     * [b274b11](https://github.com/cilium/tetragon/commit/b274b117984ba63f6c633b577ee2388f0493f88c)   (pkg/sensors/tracing/loader_windows.go)
> 
> 
> please add the commit @lmb suggested above

ah you made all those changes that in last patch.. please squash those changes into original patches